.github/workflows/test-rules.yaml

```
@@ -11,7 +11,45 @@ on:
   workflow_call:
 
 jobs:
+  extract-languages:
+    runs-on: ubuntu-latest
+    outputs:
+      languages: ${{ steps.extract.outputs.languages }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: extract
+        name: Extract languages from file
+        run: |
+          relative_path="crates/cli/src/datadog_utils.rs"
+          
+          if [ ! -f "$relative_path" ]; then
+            echo "Error: could not find target file"
+            exit 1
+          fi
+          
+          concat_languages=$(
+            # Find the definition of the slice
+            grep -Poz 'const DEFAULT_RULESETS_LANGUAGES: &\[&str\] = &\[\s*?(?:.*?\s*)+?\s*?\];' "$relative_path" |
+            # Strip the null byte added by -z
+            tr '\0' '\n' |
+            # Get any strings present
+            grep -Po '".*?"' |
+            # Delete quotation marks
+            tr -d '"' |
+            # Convert each newline to a space
+            tr '\n' ' ' |
+            # Strip a trailing space
+            sed 's/ $//'
+          )
+          
+          if [ -z "$concat_languages" ]; then
+          echo "Error: could not parse default ruleset languages"
+            exit 1
+          fi
```

Similar to the above comment about setting an error message (also, indent the echo command)

```suggestion
          if [ -z "$concat_languages" ]; then
          	echo "::error::Could not parse default ruleset languages"
            exit 1
          fi
```